### PR TITLE
WIP: ENH: 尝试用sigmod放缩std

### PR DESCRIPTION
**Pull request by facaiy**

### CODE REVIEW

File: code/fool/imagenet_gan_weight.py [hunk 1]
```diff
@@ -110,25 +111,10 @@ def calc_shuffle_loss(x):
...
-"""
-
-
-def std_loss(y_true, y_pred):
-    _ = y_pred
-    return -K.std(y_true, axis=(-3, -2, -1))
+def facai_loss(y_true, y_pred):
+    _ = y_true
+    #return K.zeros([batch_size])
+    return 1 - K.sigmoid(y_pred)
```
Comment (facaiy) [inline code/fool/imagenet_gan_weight.py]: 我把std做了sigmode变换，这样 std = 0, loss = 0.5;  std -> infinite, loss -> 0。

但迭代没有效果，都是颜色均一的图。

File: code/fool/imagenet_gan_weight.py [hunk 2]
```diff
@@ -110,25 +111,10 @@ def calc_shuffle_loss(x):
...
-   return out
-"""
-
-
-def std_loss(y_true, y_pred):
-    _ = y_pred
-    return -K.std(y_true, axis=(-3, -2, -1))
+def facai_loss(y_true, y_pred):
+    _ = y_true
+    #return K.zeros([batch_size])
```
Comment (facaiy) [inline code/fool/imagenet_gan_weight.py]: 即使把这个loss设为0，也不能复现妳当时的结果。我觉得很奇怪。

File: code/fool/imagenet_gan_weight.py (hunk 1 above)
Comment (facaiy) [reply code/fool/imagenet_gan_weight.py]: 注意，y_true，直接被忽略了

File: code/fool/imagenet_gan_weight.py [hunk 3]
```diff
@@ -98,7 +99,7 @@ def generator_containing_discriminator(g, d):
     def calc_shuffle_loss(x):
         #rx = x
         #return tf.reduce_mean(tf.square(x - rx), axis=(-3, -2, -1))
-        return -K.std(x, axis=(-3, -2, -1))
+        return K.std(x, axis=(-3, -2, -1))
```
Comment (facaiy) [inline code/fool/imagenet_gan_weight.py]: 注意，std没有取负，直接送出
